ruby/multi/Dockerfile

```diff
@@ -21,25 +21,24 @@ ENTRYPOINT /bin/bash
 #   update it to 1.3.0 once we drop Ruby 3.0 support.
 # * The bundler gem is pinned to 2.5.23 because 2.6 requires Ruby 3.1. We can
 #   update it to 2.6 once we drop Ruby 3.0 support.
-ENV RUBY_30_VERSION=3.0.7 \
-    RUBY_31_VERSION=3.1.6 \
+ENV RUBY_31_VERSION=3.1.6 \
     RUBY_32_VERSION=3.2.7 \
     RUBY_33_VERSION=3.3.7 \
     RUBY_34_VERSION=3.4.2 \
     BUNDLER_VERSION=2.5.23 \
```

Comment:
Let's update this to 2.6.8 now that we can depend on Ruby 3.1. Also remove the comment above regarding this.